File: Python/Warmup-2/array_count9.py

# this function return the number of 9´s in the given array
def array_count9(nums):
	count = 0

	for x in range(0,len(nums)):
		if nums[x] == 9:
			count += 1

	return count

print(array_count9([1, 2, 9]))
print(array_count9([1, 9, 9]))
print(array_count9([1, 9, 9, 3, 9]))

File: Java/Warmup-2/arrayCount9.java

public class warmup2 {

	// this function return the number of 9´s in the given array
	public int arrayCount9(int[] nums) {
	
		int count = 0;
		
		for (int i = 0; i < nums.length; i++) {
			if (nums[i] == 9)
				count++;
		}
		
		return count;
		
	}
	
	public static void main(String[] args) {
		
		warmup2 o = new warmup2();

		int[] x = {1, 2, 9};
		int[] y = {1, 9, 9};
		int[] z = {1, 9, 9, 3, 9};
		
		System.out.println(o.arrayCount9(x));
		System.out.println(o.arrayCount9(y));
		System.out.println(o.arrayCount9(z));

	}

}
